lib/test/evaluation_test.py

```diff
@@ -0,0 +1,31 @@
+from lib import evaluation
+import numpy as np
+import pprint
+
+class EvaluationTest:
+	
+	def __init__(self):
+		self.y_true = np.random.randint(2, size=100).reshape(-1, 1)
+		self.y_pred = np.linspace(0.0, 1.0, 100).reshape(-1, 1)
```

Comment:
Since y_pred is prediction result of classification model, the values should be 0,1, right?

Reply:
For classification model evaluation, there are overall(aggregated) evaluation and slice evaluation in our OpenAPI doc. You need to take it into consideration when you design the evaluation api as well. Please check the OpenAPI doc and following doc as references.

https://madewithml.com/courses/mlops/evaluation/